app/src/main/java/com/cricketchirps/MainActivity.java

```diff
@@ -12,6 +18,56 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        Button button = findViewById(R.id.btn_submit);
+        Button button_submit = findViewById(R.id.btn_submit);
+        final RadioButton radioBtn_celsius = findViewById(R.id.radioBtn_celsius);
+        final RadioButton radioBtn_fahrenheit = findViewById(R.id.radioBtn_fahrenheit);
+        final TextView textView_temperature = findViewById(R.id.textView_temperature);
+        final EditText txtBx_data =findViewById(R.id.txtBx_data);
+        final Toast toast_fahrenheit = Toast.makeText(getApplicationContext(),"Count the number of chirps in 14 seconds",Toast.LENGTH_SHORT);
+        final Toast toast_celcious = Toast.makeText(getApplicationContext(),"Count the number of chirps in 25 seconds",Toast.LENGTH_SHORT);
+
+        radioBtn_celsius.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                radioBtn_fahrenheit.setChecked(false);
+
+                toast_celcious.setGravity(Gravity.TOP,0,155);
+                toast_celcious.show();
+            }
+        });
+
+        radioBtn_fahrenheit.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                radioBtn_celsius.setChecked(false);
+
+                toast_fahrenheit.setGravity(Gravity.TOP,0,155);
+                toast_fahrenheit.show();
+            }
+        });
+
+        button_submit.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Integer chirpsNumber = Integer.parseInt(txtBx_data.getText().toString());
+
+                //if(txtBx_data.val) {
```

Comment:
When you create a PR for code review try to remove all you code commented, for improve our readability.